page.go

```diff
@@ -75,93 +74,114 @@ func (lp *Page) SetContent(c PageContent) {
 	lp.content = c
 }
 
-func (lp *Page) Prepare() {
-	lp.handleComponentsLifeTime()
-	lp.entry.Prepare()
-}
-
+// Call the component in sequence of life cycle
 func (lp *Page) Mount() {
-	lp.entry.Mount()
+
+	// Enable components lifecycle channel receiver
+	lp.enableComponentLifeCycleReceiver()
+
+	// pass mount live component with lifecycle channel
+	err := lp.entry.Create(lp.ComponentsLifeCycle)
+
+	if err != nil {
+		panic(err)
```

Comment:
It would be a good idea to wrap panics like this err. E.g.

`panic(fmt.Errorf("mount: create entry: %w", err)`